doc/rm/comportability_specification/index.md

```diff
@@ -504,6 +506,34 @@ More details follow.
 }
 {{< /wavejson >}}
 
+### Status Type Interrupt
+
+Status type interrupts forwards the internal status signals to the local processor.
+They may or may not be latched inside the hw module.
+The difference between the event type and status type interrupt is the persistence.
+If the caused signals are dropped by the processor or even by HW logic, the status type interrupts drop the interrupt lines. SW is responsible to mask the interrupts if they want to process the interrupt in a deferred way.
```

Comment:
could we say `the status type interrupts do not drop the interrupt lines until the cause signals are dropped by the processor or hardware logic`? Are you trying to explain the persistence does not get attached to `INTR_STATE`?

Reply:
Sounds good. Let me revise.